src/util.ts

```diff
@@ -347,8 +347,8 @@ export class Util {
         }
 
         let haveLevel = 0;
-        if (res && res.users_default) {
-            haveLevel = res.users_default;
+        if (res && res.users_default || 0) {
+            haveLevel = res.users_default || 0;
         }
         if (res && res.users && res.users[userId] !== undefined) {
```

Comment:
```suggestion
        if (res && res.users && typeof res.users[userId] === "number") {
```

Reply:
To be honest I don't know why it was failing before and it doesn't seem to fail anymore even without my commit, so I just forced pushed just this change